rasa_nlu/training_data/util.py

```diff
@@ -24,3 +25,23 @@ def check_duplicate_synonym(entity_synonyms, text, syn, context_str=""):
     if text in entity_synonyms and entity_synonyms[text] != syn:
         logger.warning("Found inconsistent entity synonyms while {0}, overwriting {1}->{2}"
                        "with {1}->{2} during merge".format(context_str, text, entity_synonyms[text], syn))
+
+
+def generate_lookup_regex(file_path, print_data_size=True):
+    """creates a regex out of the contents of a lookup table file"""
+    lookup_elements = []
+    with open(file_path, 'r') as f:
+        for l in f.readlines():
+            new_elements = [e.strip() for e in l.split(',')]
+            if '' in new_elements:
+                new_elements.remove('')
+            lookup_elements += new_elements
+    regex_string = '(?i)(' + '|'.join(lookup_elements) + ')'
```

Comment:
hmm, did we not decide on using word boundaries?

Reply:
yes, I haven't made the change yet

Reply:
fixed